dockerbuild/wait_for_mongodb.sh

```diff
@@ -14,14 +14,14 @@ until mongo "$host"/foo --eval "db.getCollectionNames()" > /dev/null 2>&1; do
     >&2 echo "MongoDB is unavailable - sleeping"
     sleep 1
 done
-
 >&2 echo "MongoDB is up - executing command"
 
+
 # Substitute the desired MongoDB hostname in the built project's conf file.
 before="$PWD"
-cd /modeldb/server
-cp src/main/resources/reference-docker.conf target/classes/reference.conf
-sed -i "s/MONGODB_HOST/$host/" target/classes/reference.conf
+
+cd /modeldb/server/codegen
```

Comment:
You shouldn't need this anymore, since you're using an absolute path below.

Reply:
removed